ONSdigital/eq-survey-runner
# Refactor alerts/errors

### Changes
- refactored response errors to use the same CSS as the alert box
- removed some unused CSS and tidied some stuff up
### How to test
- `npm run compile`
- visit a survey and submit with errors
- confirm error styling looks as expected (ie. it hasn't changed)
### Who can test

 anyone
